client/signup/steps/design-type-with-store/bluehost-store/index.jsx

```diff
@@ -11,56 +11,54 @@ import FormButton from 'components/forms/form-button';
 import StepHeader from 'signup/step-header';
 import Button from 'components/button';
 import { abtest } from 'lib/abtest';
+import { localize } from 'i18n-calypso';
 
-export default React.createClass( {
-	displayName: 'BluehostStoreStep',
+export const BluehostStoreStep = props => {
+	const { translate } = props;
 
-	redirectToPartner() {
+	const redirectToPartner = () => {
 		if ( 'bluehostWithWoo' === abtest( 'signupStoreBenchmarking' ) ) {
 			window.location.href = 'https://www.bluehost.com/web-hosting/signup?flow=woocommerce';
 		} else {
 			window.location.href = 'https://www.bluehost.com/wordpress';
 		}
-	},
+	};
 
-	getPrice() {
-		return ( 'bluehost' === abtest( 'signupStoreBenchmarking' ) ) ? '$3.95' : '$12.95';
-	},
+	const getPrice = () => ( 'bluehost' === abtest( 'signupStoreBenchmarking' ) ) ? '$3.95' : '$12.95';
```

Comment:
Can probably push this up above the component in the module space as well. I like to do this because it ends up creating something like a _static function_ instead of a _class method_ on the React components, keeping the components trim.